packages/adapter-prisma/src/adapter-prisma.js

```diff
@@ -269,20 +274,18 @@ class PrismaAdapter extends BaseKeystoneAdapter {
             `ALTER SEQUENCE \"${this.dbSchemaName}\".\"${relname}\" RESTART WITH 1;`
           );
         }
+      } else if (this.provider === 'sqlite') {
+        const tables = await this.prisma.$queryRaw(
+          "SELECT name FROM sqlite_master WHERE type='table';"
+        );
+        for (const { name } of tables) {
+          await this.prisma.$queryRaw(`DELETE FROM "${name}";`);
+        }
       } else {
-        // If we're in prototype mode then we need to rebuild the tables after a reset
-        this._runPrismaCmd(`migrate reset --force --preview-feature`);
-        await runPrototypeMigrations(this._url(), this.prismaSchema, path.resolve(this.schemaPath));
-      }
-    } else if (this.provider === 'sqlite') {
-      const tables = await this.prisma.$queryRaw(
-        "SELECT name FROM sqlite_master WHERE type='table';"
-      );
-      for (const { name } of tables) {
-        await this.prisma.$queryRaw(`DELETE FROM "${name}";`);
+        throw new Error('Only "postgresql" and "sqlite" providers are supported');
```

Comment:
Given we need to have a case for all the providers we'll want to support to run the tests fast enough, having a general case that we never actually use or test seems silly so I changed to an error.

Reply:
Oh, nice catch, this was causing our sqlite tests to be way slower than they needed to be on CI!